src/css/main.css

```diff
@@ -128,6 +128,15 @@ button:hover{
     float: left;
 }
 
+.max-runoff-volume {

```

Comment:
Thanks for bringing our attention to the fact that sometimes the maximum peak runoff and maximum peak runoff could be the same value, which causes some complexity. For the sake of simplicity, let's scrap the table highlighting. Below the "Runoff Results" table, please add a table called "Critical Durations": (fake values here)

| Maximum Runoff Value | Storm Duration (hrs) |
| ------------------------- | --------------------- | 
| Maximum Runoff Volume | 12 | 
| Maximum Peak Runoff | 3 |